src/lib/radio/radio.ts

```diff
@@ -89,11 +80,8 @@ export const _MatRadioGroupMixinBase: CanDisableCtor & typeof MatRadioGroupBase
     'role': 'radiogroup',
     'class': 'mat-radio-group',
   },
-  inputs: ['disabled'],
 })
-export class MatRadioGroup extends _MatRadioGroupMixinBase
-    implements AfterContentInit, ControlValueAccessor, CanDisable {
-
+export class MatRadioGroup implements AfterContentInit, ControlValueAccessor, CanDisable {
```

Comment:
Can you please remove `CanDisable` here? Since we don't use the mixin anymore, we don't need the interface either.